apollo-mockserver/api/apollo-mockserver.api

```diff
@@ -27,16 +17,16 @@ public final class com/apollographql/apollo3/mockserver/MockRequest {
 
 public final class com/apollographql/apollo3/mockserver/MockResponse {
 	public fun <init> ()V
-	public fun <init> (ILokio/ByteString;Ljava/util/List;Ljava/util/Map;J)V
-	public synthetic fun <init> (ILokio/ByteString;Ljava/util/List;Ljava/util/Map;JILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (ILkotlinx/coroutines/flow/Flow;Ljava/util/Map;J)V
```

Comment:
I think considering people were most probably not using chunks, this is probably ok. The main ways to construct a MockResponse with a `String` or a `ByteString` are still there.

Reply:
👍 Speaking of which.. Should we make this use the `Builder` pattern, like other parts of the API?

Reply:
I think it would make sense yeah 😁. I'll take a look.